src/disco/topo/fd_topo.h

```diff
@@ -210,6 +211,14 @@ typedef struct {
     struct {
       ulong accounts_cnt;
     } benchg;
+
+    struct {
+      char    repair_my_intake_addr[ 22 ];   // len('255.255.255.255:65535') == 22
```

Comment:
We are staying away from making config changes in these PRs, there will be a follow up for getting all the config stuff right since it might require a large refactor.

Reply:
Maybe.. but it seems simple to fix here since you're adding it. The port doesn't get parsed unless you `run` the app, which prevents `configure check` from flagging invalid ip addresses, among other issues. You can see how other IP addresses are parsed in config.c and loaded into tile config.